### ISSUE TITLE
qubes.InstallPackage service

### ISSUE THREAD

**Issue by rootkovska** [QubesOS/qubes-issues#2867]
Request installation of package (RPM, DEB, etc) in the destVM.
Note:
  - It's up to the target VM to interpret this as a command to install DEB, RPM, or whatever
  - The package must be available in an already defined repository (this call does _not_ bring custom packages)
  - This is _not_ part of the Admin API, specifically the target is handled by the destVM, not by dom0, policy does not use 'target=dom0' redirects, does not include pre-defined include/admin-api* files.

**Comment by jpouellet** [comment]
What is the intended use case?

If this is intended to be part of some larger automation where the target specified by the calling VM should be respected rather than chosen by the user, then this appears in violation of the proposed ["service-oriented qrexec"](https://github.com/QubesOS/qubes-issues/issues/910#issuecomment-275853984) model.

**Comment by marmarta** [comment]
This would be extremely useful for various config tools: instead of saying "if you want to use this, install this-and-that in the template", we could just offer a way to do it automatically. Examples: shutdown-idle in VM settings, qubes-u2f in global config.

**Comment by ben-grande** [comment]
@andrewdavidwong any label for the updater? Is it `C: Updates`? I think the same developer and tool that did the Updater will be used, if this is done.

**Comment by andrewdavidwong** [comment]
> [@andrewdavidwong](https://github.com/andrewdavidwong) any label for the updater? Is it `C: Updates`?

Yes, correct. For the GUI tool updater, there's also `C: Qubes GUI tools`.
